pkg/resource/imageconfig.go

```diff
@@ -71,9 +70,7 @@ func (gic *generatorImageConfig) Create() error {
 func (gic *generatorImageConfig) Update(o runtime.Object) (bool, error) {
 	ic := o.(*configapi.Image)
 
-	newObjectMeta := gic.objectMeta()
-
-	modified := strategy.Metadata(&ic.ObjectMeta, &newObjectMeta)
```

Comment:
Removing this line means that our operator won't restore metadata (OwnerReferences) if someone changes it.

Reply:
this was somewhat intentional.  the image config cluster object should not be owned by the image registry operator, nor tied to the image-registry resource lifecycle. It represents configuration of things beyond the registry itself and if if a user customizes it, we don't want it deleted out from under them.

i'm going to update this PR to *not* set the ownerref when we create the object.

Reply:
updated.  the idea here is that this generator is responsible for only two things:

1) creating the object if it does not exist already (it may exist because a user can create it themselves, or something else could have created it. it's a global config object).
2) updating one specific status field within the object (status.internalregistryhostname).  it is not responsible for maintaining the object metadata, it does not own those fields.